ansible.cfg

```diff
@@ -1,2 +1,5 @@
 [defaults]
 hash_behaviour=merge
+
+[ssh_connection]
+pipelining = True
```

Comment:
This doesn't look related to the kerberos changes.  It addresses another request we had regarding performance.  Are there any other changes to be aware of?